### ISSUE TITLE
Missing focus rect on logo, top nav, buttons

### ISSUE THREAD

**Issue by femmebot** [18F/nsf-sbir#688]
## Expected Behavior

- [x] Focus rect is missing on "logo", top nav ("about", "portfolio", "apply for funding"). 

- [x] Focus rect is hard to see on Yellow buttons "Start your application" and View recent awardees

see: 508 compliance report [Index N° 12, About N° 12, Portfolio N° 12, Apply N° 12](https://docs.google.com/spreadsheets/d/1yWVeVHPJtoqsCTEwnYalqDTSXNkr5E_Bn20msVmkP-8/edit#gid=0)

**Comment by femmebot** [comment]
May want to consider also tacking on #679 Indicate active/selected tab in navigation
